### ISSUE TITLE
Manifest has many repo that are not needed

### ISSUE THREAD

**Issue by rancidfrog** [OnePlusOSS/android#16]
Foe example: taking a look at manifest for op5 - QC8998_N_7.1:
It includes vendors of other devices, too,
https://github.com/OnePlusOSS/android/blob/oneplus/QC8998_N_7.1/default.xml#L515#L522

default.xml should be updated with only repos necessary and used for building for device, to follow good practice.

**Comment by ghost** [comment]
Yeah, a bunch of references to SD660 - since when does oneplus ever use anything but the 8 series chips? Lol
